.circleci/scripts/test-run-e2e-timeout-minutes.ts

```diff
@@ -1,13 +1,14 @@
 import { fetchManifestFlagsFromPRAndGit } from '../../development/lib/get-manifest-flag';
-import { filterE2eChangedFiles } from '../../test/e2e/changedFilesUtil';
+import { filterE2eChangedFiles, readChangedFiles } from '../../test/e2e/changedFilesUtil';
 
 fetchManifestFlagsFromPRAndGit().then((manifestFlags) => {
   let timeout;
 
   if (manifestFlags.circleci?.timeoutMinutes) {
     timeout = manifestFlags.circleci?.timeoutMinutes;
   } else {
-    const changedOrNewTests = filterE2eChangedFiles();
+    const changedFiles = readChangedFiles();
+    const changedOrNewTests = filterE2eChangedFiles(changedFiles);
```

Comment:
due to a change in the filterE2eChangedFiles as now it accepts the files in the func argument, so it can also be re-used in the github action job